src/parser/druid/feral/CONFIG.js

```diff
@@ -30,7 +30,7 @@ export default {
   ),
   // A recent example report to see interesting parts of the spec. Will be shown on the homepage.
   // This is the current highest 8.0.1 log for Heroic Varimathas from Antorus (a simple single target fight.) Replace with a BFA log when available.
-  exampleReport: '/report/mp7QwgKjdYqknzMT/12-Heroic+Varimathras+-+Kill+(3:11)/143-Opter',
+  exampleReport: '/report/Fypvm9C4tQ1VBzNq/34-Heroic+Vectis+-+Kill+(5:47)/32-Haramb%C3%A0e',
```

Comment:
Should remove this line 
``  // This is the current highest 8.0.1 log for Heroic Varimathas from Antorus (a simple single target fight.) Replace with a BFA log when available.
``